docs/reference/crds/verifiers.md

```diff
@@ -29,8 +29,8 @@ spec:
   parameters:
     verificationCertStores:  # certificates for validating signatures
       certs: # name of the trustStore
-        - certstore-akv # name of the certificate store CRD to include in this trustStore
-        - certstore-akv1 
+        - default/certstore-akv # namespace/name of the certificate store CRD to include in this trustStore
+        - default/certstore-akv1 
```

Comment:
Is the namespace value customizable? Apart from the `default` namespace, can users specify another namespace?